testsuite/tests/properties/is_constant_object/pack.adb

```diff
@@ -0,0 +1,9 @@
+package body Pack is
+   function My_Fun (I : Integer) return Integer is
+   begin
+      return I + 1;
+   end;
+
+   My_2 : Integer renames My_Fun (1);
```

Comment:
I don't understand how this actually tests the change you made in `CallExpr.is_constant`. Surely `My_Fun` properly resolves to a declaration in this case as well (a SubpBody to be more accurate) so `referenced_decl` should not be null.

Reply:
Here, `is_constant_object` is called on the `ObjectDecl` `My_2`, which, since it is a renames, ends to call `is_constant` on the CallExpr  name`My_Fun (1)` (calling `referenced_decl` on it returns null).

Reply:
As discussed live, looks like I missed something, I'm on it!

Reply:
Alright it looks better now!